Content.Client/Preferences/UI/HumanoidProfileEditor.xaml

```diff
@@ -84,6 +84,12 @@
                                     <Control HorizontalExpand="True"/>
                                     <Button Name="ShowClothes" Pressed="True" ToggleMode="True" Text="{Loc 'humanoid-profile-editor-clothing-show'}" HorizontalAlignment="Right" />
                                 </BoxContainer>
+                                <!-- Show loadouts -->
+                                <BoxContainer HorizontalExpand="True">
+                                    <Label Text="{Loc 'humanoid-profile-editor-loadouts'}" />
+                                    <Control HorizontalExpand="True"/>
+                                    <Button Name="ShowLoadouts" Pressed="True" ToggleMode="True" Text="{Loc 'Show'}" HorizontalAlignment="Right" />
```

Comment:
Can't you just use horizontal alignment to align each one instead of a dummy control.